schemas/sendle/pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-jstruct = ""
```

Comment:
I couldn't do `pip install -e .` on this schema package unless I changed this dependency – Poetry was telling me that it couldn't understand the dependency version. I chose `*` based on this page: https://python-poetry.org/docs/dependency-specification/#wildcard-requirements

Reply:
In fact, We have taken a step back from poetry at the moment.
If you don't mind. I would suggest replacing this with `setup.py`  instead.

Reply:
Ah, will do! Are there any examples I should follow from of courier schemas using `setup.py` without poetry?

Reply:
Yup. There you go 
https://github.com/karrioapi/karrio/blob/856e848727536763cf780606bfc887ab5104b864/schemas/dpdhl/setup.py#L1-L26

Just instead of `["six", "lxml"]`, it will be ["jstruct"]

Reply:
Done, thank you!!